wasm2c/examples/fac/fac.c

```diff
@@ -1,702 +0,0 @@
-/* Automatically generated by wasm2c */
```

Comment:
Can you restore this file?   I like to have these files checked in so that when we change the wasm2c generator we can see the effect it has on the output (as part of the diff).

Reply:
Done!